src/styles/button.scss

```diff
@@ -0,0 +1,35 @@
+.button {
+  border: none;
+  border-radius: 5px;
+  width: 100%;
+  height: 40px;
+  box-sizing: border-box;
+  transition: all 144ms;
+
+  &--buy {
+    font-family: "Roboto", Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+
+    background-color: $button-background-color;
+    color: $button-font-color;
+    // font-family: "Roboto", Arial, Helvetica, sans-serif;
+    // font-style: normal;
+    // font-weight: 700;
+    // font-size: 14px;
+    // line-height: 16px;
+
+    // text-align: center;
+    // text-transform: uppercase;
+  }
+
+  &--buy:hover {
+    border: 1px solid $button-border-color-hover;
+    background-color: $button-background-color-hover;
+    color: $button-font-color-hover;
+  }
+}
```

Comment:
No need to create separate file, and no need to add additional class, just use only one [card__button](url)
Also remove commented code